mage_ai/frontend/components/PipelineDetail/AddNewBlocks/utils.tsx

```diff
@@ -44,6 +44,7 @@ const getDataSourceTypes = (
         DataSourceTypeEnum.KINESIS,
         DataSourceTypeEnum.MONGODB,
         DataSourceTypeEnum.OPENSEARCH,
+        DataSourceTypeEnum.AZURE_DATA_LAKE,
```

Comment:
Let's sort them in alphabetical order

Reply:
sorted on 23cc8513f148e9eb7e07fd3d283fe472ea62eae5